sdk/push/src/main/java/com/klaviyo/push/KlaviyoPushService.kt

```diff
@@ -18,12 +18,11 @@ import com.klaviyo.coresdk.utils.KlaviyoPreferenceUtils
  */
 class KlaviyoPushService : FirebaseMessagingService() {
     companion object {
-        internal const val PUSH_TOKEN_PREFERENCE_KEY = "PUSH_TOKEN"
-
-        private const val REQUEST_PUSH_KEY = "\$android_tokens"
+        internal const val PUSH_TOKEN_KEY = "push_token"
+        private const val PUSH_TOKEN_APPEND_KEY = "\$android_tokens"
```

Comment:
later, we might want to consolidate these event keys into some enum somewhere else maybe, future tech debt to clean up

Reply:
Agreed, I was also thinking this might get revisited if we try to make `dataStore` internal and instead have the `push` package use some other method.

Reply:
This can get updated when I split the project up into more modules.